python/src/aac/plugins/first_party/plugin_management/plugin-management.yaml

```
@@ -0,0 +1,113 @@
+model:
+  name: plugin-management
+  description: An AaC plugin that manages which plugins available in the language context.
+  behavior:
+    - name: list-plugins
+      type: command
+      description: Display a list of available plugins.
+      input:
+        - name: --all
+          type: bool
+          python_type: bool
+          description: Display a list of all the installed AaC plugins.
+        - name: --active
+          type: bool
+          python_type: bool
+          description: Display a list of all the active AaC plugins.
+        - name: --inactive
+          type: bool
+          python_type: bool
+          description: Display a list of all the inactive AaC plugins.
+      output:
+        - name: plugins
+          type: string[]
+          python_type: list[str]
+          description: The list of plugin names.
+      acceptance:
+        - scenario: Display a list of all installed plugins.
+          when:
+            - The command is run with the {{plugin-management.input.--all}} flag.
+          then:
+            - A list of all plugins installed on the system will be displayed.
+        - scenario: Display a list of all plugins that are active on the system.
+          when:
+            - The command is run with the {{plugin-management.input.--active}} flag.
+          then:
+            - A list of all plugins installed on the system that are active will be displayed.
+        - scenario: Display a list of all plugins that are installed but not active on the system.
+          when:
+            - The command is run with the {{plugin-management.input.--inactive}} flag.
+          then:
+            - A list of all plugins installed on the system that are inactive will be displayed.
+        - scenario: Help displayed when no flags given.
+          when:
+            - The command is run without arguments.
+          then:
+            - The command help text is displayed.
+        - scenario: Error when multiple flags given.
+          when:
+            - More than one flag is provided.
+          then:
+            - An error is displayed to the user stating that flags are mutually exclusive.
+    - name: activate-plugin
+      type: command
+      description: Activate a plugin that's available and inactive on the system.
+      input:
+        - name: plugin-name
+          type: string
+          python_type: str
+          description: The name of the plugin to be activated.
+      acceptance:
+        - scenario: Plugin is activated.
+          given:
+            - The plugin named {{plugin-management.activate-plugin.plugin-name}} is installed on the system.
+            - The plugin named {{plugin-management.activate-plugin.plugin-name}} is inactive on the system.
```

It sounds like we'd want another plugin for this scenario since only the `plugin-management` plugin can install plugins.
```suggestion
            - The plugin named {{test-plugin.plugin-name}} is installed on the system.
            - The plugin named {{test-plugin.plugin-name}} is inactive on the system.
```

---

Resolving per our discussion.